<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>TransformDialogBase</class>
 <widget class="QDialog" name="TransformDialogBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MusE: Midi Input Transformator</string>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>11</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QGroupBox" name="GroupBox3" >
     <property name="title" >
      <string>Filter</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>11</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="1" column="0" >
       <widget class="QComboBox" name="selEventOp" >
        <item>
         <property name="text" >
          <string>All</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Equal</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Unequal</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QComboBox" name="selType" >
        <item>
         <property name="text" >
          <string>Note</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Poly Pressure</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Control Change</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Aftertouch</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Pitch Bend</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>NRPN</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>RPN</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="2" >
       <widget class="QLabel" name="TextLabel3" >
        <property name="text" >
         <string>Value 2</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QLabel" name="TextLabel2" >
        <property name="text" >
         <string>Value 1</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QLabel" name="TextLabel1" >
        <property name="text" >
         <string>Event Type</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QComboBox" name="selVal1Op" >
        <item>
         <property name="text" >
          <string>Ignore</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Equal</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Unequal</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Higher</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Lower</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Inside</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Outside</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="1" >
       <widget class="QSpinBox" name="selVal1b" >
        <property name="maximum" >
         <number>127</number>
        </property>
       </widget>
      </item>
      <item row="3" column="2" >
       <widget class="QSpinBox" name="selVal2b" >
        <property name="maximum" >
         <number>127</number>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QSpinBox" name="selVal1a" >
        <property name="maximum" >
         <number>127</number>
        </property>
       </widget>
      </item>
      <item row="1" column="2" >
       <widget class="QComboBox" name="selVal2Op" >
        <item>
         <property name="text" >
          <string>Ignore</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Equal</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Unequal</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Higher</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Lower</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Inside</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Outside</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="3" >
       <widget class="QComboBox" name="selChannelOp" >
        <item>
         <property name="text" >
          <string>Ignore</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Equal</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Unequal</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Higher</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Lower</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Inside</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Outside</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="3" >
       <widget class="QLabel" name="TextLabel2_2" >
        <property name="text" >
         <string>Channel</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2" >
       <widget class="QSpinBox" name="selVal2a" >
        <property name="maximum" >
         <number>127</number>
        </property>
       </widget>
      </item>
      <item row="2" column="3" >
       <widget class="QSpinBox" name="selChannelVala" >
        <property name="maximum" >
         <number>127</number>
        </property>
       </widget>
      </item>
      <item row="3" column="3" >
       <widget class="QSpinBox" name="selChannelValb" >
        <property name="maximum" >
         <number>127</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="GroupBox5" >
     <property name="title" >
      <string>Processing</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>11</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="0" column="0" >
       <widget class="QLabel" name="TextLabel5" >
        <property name="text" >
         <string>Event Type</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QComboBox" name="procEventOp" >
        <item>
         <property name="text" >
          <string>Keep</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Fix</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QComboBox" name="procType" >
        <item>
         <property name="text" >
          <string>Note</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Poly Pressure</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Control Change</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Aftertouch</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Pitch Bend</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>NRPN</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>RPN</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QComboBox" name="procVal1Op" >
        <item>
         <property name="text" >
          <string>Keep</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Plus</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Minus</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Multiply</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Divide</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Fix</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Value 2</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Invert</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>ScaleMap</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Flip</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Dyn</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Random</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QLabel" name="TextLabel6" >
        <property name="text" >
         <string>Value 1</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" >
       <widget class="QLabel" name="TextLabel7" >
        <property name="text" >
         <string>Value 2</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" >
       <widget class="QSpinBox" name="procVal1b" >
        <property name="maximum" >
         <number>9999</number>
        </property>
       </widget>
      </item>
      <item row="3" column="2" >
       <widget class="QSpinBox" name="procVal2b" >
        <property name="maximum" >
         <number>9999</number>
        </property>
       </widget>
      </item>
      <item row="3" column="3" >
       <widget class="QSpinBox" name="procChannelValb" >
        <property name="maximum" >
         <number>127</number>
        </property>
       </widget>
      </item>
      <item row="1" column="2" >
       <widget class="QComboBox" name="procVal2Op" >
        <item>
         <property name="text" >
          <string>Keep</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Plus</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Minus</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Multiply</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Divide</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Fix</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Value 1</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Invert</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Dyn</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Random</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="3" >
       <widget class="QComboBox" name="procChannelOp" >
        <item>
         <property name="text" >
          <string>Keep</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Plus</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Minus</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Multiply</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Divide</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Fix</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Value 1</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Invert</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Dyn</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Random</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="3" >
       <widget class="QLabel" name="TextLabel2_2_2" >
        <property name="text" >
         <string>Channel</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" >
       <widget class="QComboBox" name="funcOp" />
      </item>
      <item row="2" column="1" >
       <widget class="QSpinBox" name="procVal1a" />
      </item>
      <item row="2" column="2" >
       <widget class="QSpinBox" name="procVal2a" />
      </item>
      <item row="2" column="3" >
       <widget class="QSpinBox" name="procChannelVala" />
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <tabstops>
  <tabstop>selEventOp</tabstop>
  <tabstop>selType</tabstop>
  <tabstop>selVal1Op</tabstop>
  <tabstop>selVal1a</tabstop>
  <tabstop>selVal1b</tabstop>
  <tabstop>selVal2Op</tabstop>
  <tabstop>selVal2a</tabstop>
  <tabstop>selVal2b</tabstop>
  <tabstop>selChannelOp</tabstop>
  <tabstop>selChannelVala</tabstop>
  <tabstop>selChannelValb</tabstop>
  <tabstop>procEventOp</tabstop>
  <tabstop>procType</tabstop>
  <tabstop>procVal1Op</tabstop>
  <tabstop>procVal1b</tabstop>
  <tabstop>procVal2Op</tabstop>
  <tabstop>procVal2b</tabstop>
  <tabstop>procChannelOp</tabstop>
  <tabstop>procChannelValb</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
